app/services/state_file/reminder_to_finish_state_return_service.rb

```diff
@@ -3,15 +3,13 @@ class ReminderToFinishStateReturnService
     def self.run
       cutoff_time_ago = 12.hours.ago
       batch_size = 10
-      intakes_with_no_submission = StateFileAzIntake.where("df_data_imported_at < ?", cutoff_time_ago)
-                                                    .left_joins(:efile_submissions)
-                                                    .where(efile_submissions: { id: nil })
-                                                    .where.not("state_file_az_intakes.message_tracker #> '{messages.state_file.finish_return}' IS NOT NULL")
+      intakes_with_no_submission = StateFileAzIntake.where('state_file_az_intakes.created_at < ?', cutoff_time_ago)
```

Comment:
Should consider doing the same iteration over the state intake class names as in #4475 and having `#{base_class.underscore.pluralize}` instead of separate steps for AZ & NY. Would be more consistent and any changes would be applied in one place rather than two.

Reply:
Thanks for raising, this was the first one that was done, so the latter ones have improvements.  I'll update!